spec/models/user_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) do
+    User.create(
+      name:     'Joe',
+      email:    'joe@mail.com',
+      password: 'joe_pass',
+      password_confirmation: 'joe_pass'
+    )
+  end
+
+  it 'is valid with a name, email, and password' do
```

Comment:
text inside must start with should, example it 'should be valid with a name, email, and password' .

Reply:
fix

Reply:
but Rubocop say: Do not use should when describing your tests